docs/usage/12-openapi.md

```diff
@@ -163,3 +163,17 @@ app = Starlite(
     openapi_config=OpenAPIConfig(openapi_controller=MyOpenAPIController),
 )
 ```
+
+You can override the `root` handler to serve other included UIs or your own [template](./15-templating.md):
```

Comment:
Please explain about the different render methods available.

Also -what if users want to have both swaggerUI and redoc?

Reply:
How do you feel about this: 

> The root handler serves Redoc by default via `render_redoc`. You can override the `root` handler to serve other included documentation, such as Swagger UI by returning `render_swagger_ui` or Stoplight Elements with `render_stoplight_elements`. You can also have it serve your own [template](./15-templating.md):

Do you think it should include the part about rendering templates? I'm not sure if it's relevant, but just wanted to highlight the flexibility.

Reply:
Sure thats fine